android/engine/src/main/java/org/smartregister/fhircore/engine/domain/model/OverflowMenuItemConfig.kt

```diff
@@ -17,16 +17,29 @@
 package org.smartregister.fhircore.engine.domain.model
 
 import kotlinx.serialization.Serializable
+import org.smartregister.fhircore.engine.configuration.navigation.MenuIconConfig
+import org.smartregister.fhircore.engine.util.extension.interpolate
 
 @Serializable
 data class OverflowMenuItemConfig(
   val id: Int = 1,
   val title: String = "",
   val confirmAction: Boolean = false,
-  val titleColor: String? = null,
+  val icon: MenuIconConfig? = null,
```

Comment:
Do these icons have a default size?  This will come in handy when the uploaded icons are too big